두-개-뽑아서-더하기.js

```diff
@@ -0,0 +1,14 @@
+function solution(numbers) {
+    let answer = [];
+    for (let i = 0; i < numbers.length; i++) {
+        for (let j = i + 1; j < numbers.length; j++) {
+            answer.push(numbers[i] + numbers[j]);
+        }
+    }
+    answer = answer.filter((item, index) => answer.indexOf(item) === index);
+    answer.sort((a, b) => {
+        if (a > b) return 1;
+        else return -1;
+    });
+    return answer;
```

Comment:
```suggestion
    return [...new Set(result)].sort((a, b) => a - b);
```
저 로직을 저렇게 한줄로 표현이 가능한데 자바스크립트의 Set 타입과 스프레드 연산자를 알아보시면 좋을거같아요.